README.md

```diff
@@ -117,6 +117,62 @@ cron.schedule('* * * Jan,Sep Sun', function(){
 });
 ```
 
+## Cron methods
+
+### Schedule
+
+Schedules given task to be executed whenever the cron expression ticks.
+
+Arguments:
+
+- !string expression - Cron expression
+- !Function func - Task to be executed
+- boolean? immediateStart - Whether to start scheduler immediately after create.
```

Comment:
It doesn't mean to execute the task immediately, as it says:

> Whether to start scheduler immediately after create.

It will start updating cron until the time comes to execute the task.